migrate_nidirect_node/migrate_nidirect_node_article/config/install/migrate_plus.migration.node_article.yml

```diff
@@ -21,7 +21,45 @@ source:
   node_type: article
   batch_size: 1000
 process:
-  nid: tnid
+  nid:
+    -
+      plugin: skip_on_value
+      method: row
+      source: tnid
+      value:
```

Comment:
Extracted these by cross-checking the titles of content returned by the RSS feeds + interrogating the body field of the D7 database, checking for content that had the 'Provided by the Money Advice Service' footer.